python/util/roles.py

```diff
@@ -52,7 +52,7 @@
 
 DESIGN_SECTION = {ADMIN, MANAGER, DESIGNER, CLIENT, PUBLIC}
 PRODUCTION_SECTION = {ADMIN, MANAGER, SUPPLIER}
-SHIPPING_SECTION = {ADMIN, MANAGER, DESIGNER, CLIENT}
```

Comment:
this change might have other side effects which might break things

Reply:
which type of things? didn't see any failing tests

Reply:
after a quick global search for 'SHIPPING_SECTION'  and consequently 'can_view_shipping_section', I see that this change shouldn't have any negative side effects.